2.Arrays/arrays.js

```diff
@@ -0,0 +1,102 @@
+function isArray1(arr) {
+	return arr instanceof Array;
+}
+
+console.log(isArray1([3,null,undefined,4]));
+
+function isArray2(arr) {
+	return Array.isArray(arr);
+}
+
+console.log(isArray2([3,null,undefined,4]));
+console.log(isArray2({0:2, 1:3}));
+
+function range(from, to, step) {
```

Comment:
cases should work:
range(10) [0..9]
range(1, 10) [1..9]
range(1, 10, 3) [1, 4, 7]
range(10, null, 3) [0, 3, 6, 9]
range(10, null) the same with range(10)
range(-2, -5) []
range(-5, -2) [-5, -4, -3]
range(-5, -2, 2) [-5, -3]
range(-5, -2, -2) []
range(-5, null, 2) [] 
range(-5, null, -2) [0, -2, -4]
range(-10, -20, -5) [-10, -15]
range(-20, -10, -5)
currently I have infinity loop for some cases